### PR TITLE
API Remodel

### PR DESCRIPTION
**Pull request by jordan-rash**
Makes it so that the API is more scopeable(TM) by namespace

### CODE REVIEW

File: api/nodecontrol/client.go [hunk 1]
```diff
@@ -122,7 +107,7 @@ func (c *ControlAPIClient) DirectPing(nodeId string) (*nodegen.NodePingResponseJ
 }
 
 func (c *ControlAPIClient) FindWorkload(_type, namespace, workloadId string) (*nodegen.WorkloadPingResponseJson, error) {
-	msg, err := c.nc.Request(models.AgentPingWorkloadRequestSubject(_type, namespace, workloadId), nil, DefaultRequestTimeout)
+	msg, err := c.nc.Request(models.WorkloadPingRequestSubject(_type, namespace, workloadId), nil, DefaultRequestTimeout)
```
Comment (autodidaddict) [inline api/nodecontrol/client.go:110]: If we're actually using this variable, we should remove the `_` ... Weird that the linter doesn't complain about this when it's so cranky about the inverse.

File: api/nodecontrol/client.go [hunk 2]
```diff
@@ -175,7 +159,7 @@ func (c *ControlAPIClient) DeployWorkload(namespace, nodeId string, req nodegen.
 		return nil, err
 	}
 
-	msg, err := c.nc.Request(models.DeployRequestSubject(namespace, nodeId), req_b, DefaultRequestTimeout)
+	msg, err := c.nc.Request(models.DeploySubject(nodeId), req_b, DefaultRequestTimeout)
```
Comment (autodidaddict) [inline api/nodecontrol/client.go:162]: should we call this a "direct deploy" to contrast against the "auction deploy"

Comment (autodidaddict) [reply api/nodecontrol/client.go:162]: could even change the subject from `DEPLOY` to `DDEPLOY` to contrast `ADEPLOY`

Comment (jordan-rash) [reply api/nodecontrol/client.go:162]: I can do that...

File: api/nodecontrol/client.go (hunk 1 above)
Comment (jordan-rash) [reply api/nodecontrol/client.go:110]: its because `type` is a reserved word.....I can call it something else

Comment (autodidaddict) [reply api/nodecontrol/client.go:110]: oh, hah.. makes sens. yeah, maybe `workloadType`

**Review by autodidaddict — APPROVED** [review]

**Pull request merged**